src/components/ScoobieLink.tsx

```diff
@@ -6,14 +6,33 @@ import { isExternalHref } from '../private/url';
 import { InternalLink } from './InternalLink';
 
 export const ScoobieLink = makeLinkComponent(
-  ({ children, href, ...restProps }, ref) =>
-    isExternalHref(href) ? (
-      <a rel="noreferrer" target="_blank" {...restProps} href={href} ref={ref}>
-        {children}
-      </a>
-    ) : (
+  ({ children, href, download, ...restProps }, ref) => {
+    if (isExternalHref(href)) {
+      return (
+        <a
+          rel="noreferrer"
+          target="_blank"
+          {...restProps}
+          href={href}
+          ref={ref}
+        >
+          {children}
+        </a>
+      );
+    }
+
+    if (download) {
+      return (
+        <a rel="noreferrer" {...restProps} href={href} ref={ref}>
+          {children}
```

Comment:
Might not be too bad but not sure how I can clean up this up. Repeating itself a little here. 

The externalHref anchor tag has target attribute whereas this download anchor tag doesnt